src/openrct2-ui/input/KeyboardShortcuts.cpp

```diff
@@ -146,35 +146,35 @@ bool KeyboardShortcuts::Save()
 void KeyboardShortcuts::Set(int32_t key)
 {
     // Unmap shortcut that already uses this key
-    int32_t shortcut = GetFromKey(key);
-    if (shortcut != SHORTCUT_UNDEFINED)
+    auto shortcut = GetFromKey(key);
+    if (shortcut != Input::Shortcut::Undefined)
     {
-        _keys[shortcut] = SHORTCUT_UNDEFINED;
+        _keys[static_cast<size_t>(shortcut)] = static_cast<uint16_t>(Input::Shortcut::Undefined);
```

Comment:
`static_cast<uint16_t>(Input::Shortcut::Undefined)` this isn't correct. Well it is correct in terms of what it does but it's not correct because it should be `Input::ScanCodeUndefined`